deploy-service/teletraanservice/bin/server.yaml

```diff
@@ -13,10 +13,17 @@ server:
   requestLog:
     appenders:
       - type: file
-        threshold: ALL
-        archive: false
+        archive: true
+        archivedLogFilenamePattern: /tmp/teletraan/access-%d.log
+        archivedFileCount: 5
         currentLogFilename: /tmp/teletraan/access.log
-        timeZone: UTC
+        layout:
+          type: access-json
+          timestampFormat: "yyyy-MM-dd'T'HH:mm:ss.SSSZ"
+          requestAttributes: [name]
+          customFieldNames:
+            requestAttributes: principal
+          includes: [timestamp, remoteAddress, protocol, method, requestUri, statusCode]
```

Comment:
Can we add environment type in the log message?

Example of environments and environments types:
- Production:  `env: production`
- Stage: `env: stage`
- Dev: `env: dev`

And specify environment typ as an environment variable during deployment of the Teletraan service, like `export ENV=production` which will make easier to separate logs from different environments in Kibana.

Reply:
We have other mechanisms for separating them in Kibana, such as `o11y_hostname`.